README.md

```diff
@@ -12,3 +12,22 @@ Here we document our workflows, styleguides and patterns.
 
 * [Frontend Styleguide](https://github.com/GeekHunter-Brasil/hello-world/tree/master/frontend-styleguide)
 * [Backend Styleguide](https://github.com/GeekHunter-Brasil/hello-world/tree/master/backend-styleguide)
+* [Development Styleguide](https://github.com/GeekHunter-Brasil/hello-world/tree/master/development-styleguide)
+
+
+# The Geek Manifest
+
+- No code is better than no code (if I didn't know how to program, how would I solve it?);
+- Delivered is when we can prove that the customer received value in production;
+  - Customer used;
+  - We collect usage behavior, we prove it works;
+  - We monitor bugs and performance issues to make sure it's done right;
+- TDD is an implicit requirement of any code intervention;
+- The sprint is always the guide, tasks outside of it should be the exception (“Top! Tell our PM to prioritize for us!”)
+- Refactoring code is important, but the focus always needs to be on the task
+  - fixing code outside of it can generate value but can compromise your schedule related to what you expected to deliver initially.
```

Comment:
Just to keep the same pattern

```suggestion
  - Fixing code outside of it can generate value but can compromise your schedule related to what you expected to deliver initially.
```